payments_service/braintree/templates/braintree/emails/subscription_charged_successfully.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1, maximum-scale=1">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <link rel="stylesheet" href="/dist/main.css">
+    <title>Payments UI</title>
+  </head>
```

Comment:
Not using a template for this, because using premailer as pre-build step fails on this.